operator/src/main/resources/scripts/utils.sh

```diff
@@ -720,61 +686,52 @@ function adjustPath() {
 
 #
 # checkAuxiliaryImage
-#   purpose: If the AUXILIARY_IMAGE_PATH directory exists, it echoes the contents of output files
-#            in ${AUXILIARY_IMAGE_PATH}/auxiliaryImagetLogs dir. It returns 1 if a SEVERE message
-#            is found in any of the output files in ${AUXILIARY_IMAGE_PATH}/auxiliaryImageLogs dirs.
-#            It also returns 1 if 'successfully' message is not found in the output files
-#            or if the AUXILIARY_IMAGE_PATH directory is empty. Otherwise it returns 0 (success).
+#   purpose: If the AUXILIARY_IMAGE_MOUNT_PATH directory exists, it echoes the contents of output files
+#            in ${AUXILIARY_IMAGE_MOUNT_PATH}/auxiliaryImagetLogs dir. It returns 1 if a SEVERE message
+#            is found in any of the output files in ${AUXILIARY_IMAGE_MOUNT_PATH}/auxiliaryImageLogs dirs.
+#            It also returns 1 if 'successfully' message is not found in the output files.
+#            Otherwise it returns 0 (success).
 #            See also 'auxImage.sh'.
 #            See also initAuxiliaryImage in 'utils_base.sh'.
 #
 function checkAuxiliaryImage() {
   # check auxiliary image results (if any)
-  if [ -z "$AUXILIARY_IMAGE_PATHS" ]; then
+  if [ -z "$AUXILIARY_IMAGE_MOUNT_PATH" ]; then
     trace FINE "Auxiliary Image: Skipping auxiliary image checks (no auxiliary images configured)."
     return
   fi
 
-  trace FINE "Auxiliary Image: AUXILIARY_IMAGE_PATHS is '$AUXILIARY_IMAGE_PATHS'."
-  for AUXILIARY_IMAGE_PATH in ${AUXILIARY_IMAGE_PATHS/,/ }; do
-    trace FINE "Auxiliary Image: AUXILIARY_IMAGE_PATH is '$AUXILIARY_IMAGE_PATH'."
-    traceDirs $AUXILIARY_IMAGE_PATH
-    touch ${AUXILIARY_IMAGE_PATH}/testaccess.tmp
-    if [ $? -ne 0 ]; then
-      trace SEVERE "Auxiliary Image: Cannot write to the AUXILIARY_IMAGE_PATH '${AUXILIARY_IMAGE_PATH}'. " \
-                   "This path is configurable using the domain resource 'spec.auxiliaryImageVolumes.mountPath' " \
-                   "attribute." && return 1
-    fi
-    rm -f ${AUXILIARY_IMAGE_PATH}/testaccess.tmp || return 1
+  trace FINE "Auxiliary Image: AUXILIARY_IMAGE_MOUNT_PATH is '$AUXILIARY_IMAGE_MOUNT_PATH'."
+  traceDirs $AUXILIARY_IMAGE_MOUNT_PATH
```

Comment:
traceDirs requires specifying "before" or "after" as its second parameter

(this seems to be a common mistake -- maybe fix 'traceDirs' itself so that it still works if only one parm is passed in)